src/components/nodes/File.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="q-px-md q-pt-md">
+    <q-input @input="val => { file = val[0] }"
+      filled
```

Comment:
What do you guys say about making this `borderless` instead of `filled`?

Reply:
I think it actually looks better borderless

Reply:
I think we should restyle the whole thing entirely, and also make it possible to drag and drop a file into some area. If I remember correctly this is not as difficult as it may sound.